src/components/ha-target-picker.ts

```diff
@@ -237,8 +237,6 @@ export class HaTargetPicker extends LitElement {
           : html`<span role="gridcell">
               <ha-icon-button
                 class="expand-btn mdc-chip__icon mdc-chip__icon--trailing"
-                tabindex="-1"
-                role="button"
```

Comment:
@bramkragten I'm not sure what effect was sought here, but I'm removing these attributes here as they effectively do nothing (`ha-icon-button` is already a button and will always be in the tab sequence).  Target picker needs work per #14367 anyway.

Reply:
Okay to merge before this gets conflicted?